### DISCUSSION TITLE
Alist下载文件

### DISCUSSION THREAD

**Discussion by yuhuan1126** [AlistGo/alist#1971] [Q&A]
我通过花生壳挂了内网的alist，这样我的alist就有了公网域名，但是在公网上无法下载文件，怎么解决？（看了文档但是看不懂qw）

挂下域名 https://438vw37053.zicp.fun/

**Comment by TFTree** [comment]
好像在你的域名能下载，就是速度很慢，100kb每秒，可能走的是你的流量而不是网盘的流量

**Reply by yuhuan1126** [reply]
我这在域名上下载不了，显示当前无法使用此页面

**Comment by 12189108** [comment]
下载链接有问题，一般为域名/d/文件路径，你这怎么变成了域名/p/文件路径，别乱写下载代理URL

**Reply by yuhuan1126** [reply]
我就没改下载，一直是本地代理啊[kk]

**Reply by yuhuan1126** [reply]
对了，直接点下载是p，右键文件点下载就是d
